Makefile

```diff
@@ -63,7 +63,7 @@ ROLES = lnls-ans-role-cs-studio \
 	lnls-ans-role-java \
 	lnls-ans-role-nfsclient \
 	lnls-ans-role-zabbix \
-	lnls-ans-role-nfsserver \
```

Comment:
Why are we removing nfsserver from the Makefile? Doesn't it still exists? Maybe we should remove the "nfsserver" role altogether later, but for now we can keep this unchanged I guess.

Reply:
I did not remove the role. just renamed it.